how-to-pitch/readme.md

```diff
@@ -0,0 +1,49 @@
+> Goal of this guide: Helping you to pitch
+
+# How to pitch
+
+"Trust me I got this" is one of the least reassuring sentences in the world. So how do you convey that you can do something without clearly saying that you "got diss"?
+
+<img src="https://media.giphy.com/media/XfxbrDXAkIqFa/giphy.gif" width="100%">
```

Comment:
I don't think it is :( also because of its size – at this size i'd say it's better if it wasn't there. maybe bring it back to 50%?